jetty-io/src/main/java/org/eclipse/jetty/io/ssl/SslConnection.java

```diff
@@ -1159,6 +1180,10 @@ public boolean flush(ByteBuffer... appOuts) throws IOException
                     }
                     catch (Throwable x)
                     {
+                        // Clear the encrypted output buffer so that
+                        // releaseEncryptedOutputBuffer() in the finally block
+                        // will not leak it.
+                        BufferUtil.clear(_encryptedOutput);
```

Comment:
Ditto.  Use a method that clearly always releases the buffers and then you don't need the comment explanation.

Reply:
done.